examples/lang2/erc721/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+[package]
+name = "erc721"
+version = "0.1.0"
+authors = ["XatMassacrE"]
+edition = "2018"
+
+[dependencies]
+ink_abi = { path = "../../../abi", default-features = false, features = ["derive"], optional = true }
+ink_core = { path = "../../../core", default-features = false }
+ink_lang2 = { path = "../../../lang2", default-features = false }
```

Comment:
Please make use of the new `ink_prelude` crate as has been done in other parts of ink! and other examples.